client-config/src/main/java/com/palantir/conjure/java/client/config/ClientConfiguration.java

```diff
@@ -101,6 +101,13 @@
      */
     Duration backoffSlotSize();
 
+    /**
+     * Disables the client-side sympathetic QoS. Consumers should almost always set this value to its default of false,
+     * except where there are known issues with the QoS interaction. Please consult project maintainers if flipping
+     * this value to true.
+     */
+    boolean forExpertsOnlyDisableSympatheticClientQoS();
```

Comment:
Rather than using a boolean, what about an enum something like the following:

```
enum ClientSideQoS {
  ENABLE_RECOMMENDED, DANGEROUS_DISABLE_CLIENT_QOS
}
``` 

This would mean you don't need the `private static final boolean DEFAULT_DISABLE_SYMPATHETIC_QOS = false;` thing below?